Step-by-step-Tutorials/Unit-Testing-Grains.md

```diff
@@ -69,7 +69,7 @@ namespace Tests
 
 ```
 
-As you can see having `ClassCleanup()` is optional but it's good to have it around. 
+Make sure to add `async` to the generated test method for the test to be recognized by Visual Studio. As you can see having `ClassCleanup()` is optional but it's good to have it around. 
```

Comment:
do you want to update it to the suggested sentence? Otherwise I can merge and do it myself afterwards